oslabs-beta/Kubiquity
# Add new MemoryBarChart component

WHAT DOES THIS PR DO? 
1. Adds a new chart library to our app. 
2. Removes old bar chart. 
3. Creates new bar chart. 
4. Imports data to new bar chart. 

HOW TO TEST THIS PR:
1. Run the following command: `gh pr checkout 11 && npm install && npm run dev`
2. Verify that you can see the new bar chart. 

ADDITIONAL NOTES: 
1. I thought the old bar chart looked really basic and boring, so I could a pretty easy-to-use package to give us a better looking bar chart. 
2. The new component has some built in download features that I haven't yet tested. TK
3. At this point, we probably have a few extra packages that we're not using. Trimming our packages to what's actually being used should be done, but definitely can. be like the last thing we do.
